### PR TITLE
Python module conversion

### PR DESCRIPTION
**Pull request by gamikun**
I converted file in a python module, so it can be submitted to PyPi.org
Also added code for preventing break old structure the way was used.

    from bplist import *

in __ ini __.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+setup(
+  name = 'bplist',
+  packages = ['bplist'],  
+  version = '0.1',
+  description = 'Apple binary Property List reader/writer for Python',
+  install_requires=[],
+  author = 'Vladimir Pouzanov',
+  author_email = 'farcaller@gmail.com',
+  url = 'https://github.com/gamikun/bplist-python',
```
Comment (rilian) [inline setup.py]: github.com/farcaller
;)

Comment (gamikun) [inline setup.py]: That's because I published some months ago, but i decided to make the pull request today. I just fixed it.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+  name = 'bplist',
+  packages = ['bplist'],  
+  version = '0.1',
```
Comment (farcaller) [inline setup.py]: You can safely tag this as 1.0, it didn't change in years and is extremely stable (as in, no changes are coning up)

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  description = 'Apple binary Property List reader/writer for Python',
+  install_requires=[],
+  author = 'Vladimir Pouzanov',
+  author_email = 'farcaller@gmail.com',
+  url = 'https://github.com/farcaller/bplist-python',
+  keywords = ['bplist'], 
+  classifiers = [],
```
Comment (farcaller) [inline setup.py]: Needs `license` key.

**Comment by gamikun** [comment]
I made changes.

**Comment by farcaller** [comment]
Thanks for your contribution!

**Pull request merged**